src/tracepoint_format_parser.cpp

```diff
@@ -109,18 +105,15 @@ bool TracepointFormatParser::parse(ast::Program *program, BPFtrace &bpftrace)
           std::ifstream format_file(format_file_path.c_str());
           if (format_file.fail())
           {
-            // Errno might get clobbered by bpftrace.error and bpftrace.warning.
+            // Errno might get clobbered by LOG().
             int saved_errno = errno;
 
-            bpftrace.error(std::cerr,
-                           ap->loc,
-                           "tracepoint not found: " + category + ":" +
-                               event_name);
+            LOG(ERROR, ap->loc, std::cerr)
+                << "tracepoint not found: " << category << ":" << event_name;
             // helper message:
             if (category == "syscall")
-              bpftrace.warning(std::cerr,
-                               ap->loc,
-                               "Did you mean syscalls:" + event_name + "?");
+              LOG(WARNING, ap->loc, std::cerr)
+                  << "Did you mean syscalls:" << event_name << "?";
             if (bt_verbose) {
               // Having the location info isn't really useful here, so no
               // bpftrace.error
```

Comment:
I guess you are thinking to fix this in another PR, but I think we can use `LOG(DEBUG, std::cerr)` here.